### PR TITLE
Create trie.py

### PR DESCRIPTION
**Pull request by rghvsh**
# PR Description

Adds trie data structure

## Related Issue

Closes: #62 

## Changes Made

List the changes you made in this pull request:

- Adds trie data structure.

## Author

Raghav Sharma (@rghvsh )

### CODE REVIEW

File: pyds_a/data_structures/trie.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+"""
+Author: Raghav Sharma(rghvsh)
+
+Implements trie data structure and adds search
+"""
+
+class trie:
+
```
[the remark this answers is not in the document]
Comment (rghvsh) [reply pyds_a/data_structures/trie.py:8]: sure

**Comment by rghvsh** [comment]
Hey @Tinny-Robot made some change. Is

**Comment by rghvsh** [comment]
Hey

**Comment by rghvsh** [comment]
Hey! @Tinny-Robot checked the other files and made the file analogous , changed its file structure. Is this better?

**Review by matt-wisdom — CHANGES_REQUESTED** [review]
Hello, there seems to be indentation issues in your code, please ensure that the code is properly indented (make sure the docstrings of each method are on the same indentation level as the code).

**Comment by rghvsh** [comment]
fixed indentation

**Pull request merged**